### ISSUE TITLE
why are additional type libraries being installed?

### ISSUE THREAD

**Issue by ganySA** [telerik/kendo-angular#1863]
When using @progress/kendo-angular-dateinputs - the installation installs numerous additional type libraries which i am trying to understand. I only picked this up because they cause compilation errors with Angular 7.0 but with additional inspection, why are they there in the first place?

       
```
         
        "@types/core-js": "^2.5.0",
        "@types/es6-shim": "^0.31.36",
        "@types/fancybox": "^2.1.32",
        "@types/highcharts": "^5.0.21",
        "@types/jasmine": "^2.8.6",
        "@types/jquery": "^2.0.42",
        "@types/jqueryui": "^1.12.2",
        "@types/kendo-ui": "^2018.1.2",
        "@types/microsoft-ajax": "0.0.33",


```
What is the purpose of these libraries. Specifically the Jquery libraries? highcharts? fancybox? MS ajax??

Thanks

**Comment by ganySA** [comment]
i thought so. thanks! I'll let you close the issue when the patch is in.

**Comment by tsvetomir** [comment]
Fixed in v3.5.1
